FMR/iocBoot/iocFMR-IOC-01/config.xml

```diff
@@ -6,6 +6,7 @@
             <macro name="LVDCOM_OPTIONS" pattern="^\d?\d?$" description="Options that define how the VI is started. Add selected options together to make the final value. 1 - warn if idle, 2 - start if idle, 4 - stop VIs if started on exit, 8 - stop VI on exit. (Default 1)" hasDefault="YES" defaultValue="1"/>
             <macro name="LVDCOM_HOST" pattern="^.*$" description="Host on which the vi is running, blank for localhost (Default '')" hasDefault="YES" defaultValue=""/>
             <macro name="PATH_TO_VI" pattern="^.*$" description="Full VI path including filename (Default '')" hasDefault="YES" defaultValue=""/>
+            <macro name="VNSTANDALONE" pattern="^.*$" description="Whether or not to control JUST the VNA or the VNA and other kit - YES or blank, default is blank" hasDefault="YES" defaultValue=""/>
```

Comment:
Default stated here doesn't seem to match the default in the `st.cmd` (which is blank)?